templates/tfengine/components/project/gke_clusters/main.tf

```diff
@@ -12,19 +12,21 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */ -}}
 
-{{range get . "GKE_CLUSTERS"}}
+{{- range get . "GKE_CLUSTERS"}}
+
+{{- $region := get . "REGION"}}
+{{- if not $region}}
+{{- $region = $.GKE_CLUSTER_REGION}}
```

Comment:
Could this be the "default" value in the new get? i.e. just
```
{{- $region := get . "REGION" $.GKE_CLUSTER_REGION }}
```

Reply:
I actually think it's fine to not require the top-level GKE_CLUSTER_REGION. I was hoping that if not provided, the templating engine wouldn't crash.

Reply:
I think we're going to see this pattern happening in a few places when it comes to dealing with locations/regions, so I prefer your earlier suggestion for now because it makes the templates way simpler (everything can be concisely put in one line rather than split up and using 3-4 lines). The templates are getting more complex than I was hoping for so I want to keep them simpler as much as we can for now at a slight user inconvenience.

Plus, given it is required if there is no override means we would have to document that behaviour anyway.